src/app/components/Transaction/index.tsx

```diff
@@ -42,6 +44,14 @@ type TransactionDictionary = {
 interface TransactionProps {
   referenceAddress: string
   transaction: transactionTypes.Transaction
+  network: NetworkType
+}
+
+const getBlockExplorerHref = network => {
+  const subdomain = network === 'testnet' ? 'testnet.' : ''
+  return isOasisScanEnabled()
+    ? `https://${subdomain}oasisscan.com/transactions/`
+    : `https://${subdomain}oasismonitor.com/operation/`
```

Comment:
I think this would fit better in https://github.com/oasisprotocol/oasis-wallet-web/blob/b264e85094407a02f3907ad8fb2ff04721b924ea/src/config.ts#L3-L21

Reply:
updated

for local I am returning empty strings, not sure it's Ok

Reply:
Maybe `http://localhost:9001/data/transactions?operation_id=` :shrug:

Reply:
That could work with `http://localhost:9001/data/transactions?operation_id={{txHash}}` +  `blockExplorerLink.replace('{{txHash}}', encodeURIComponent(transaction.hash))`, instead of appending /${encodeURIComponent(transaction.hash)}

Reply:
done